port_ocean/clients/port/utils.py

```diff
@@ -16,7 +16,7 @@
 # period of time, before raising an exception.
 # The max_connections value can't be too high, as it will cause the application to run out of memory.
 # The max_keepalive_connections can't be too high, as it will cause the application to run out of available connections.
-PORT_HTTP_MAX_CONNECTIONS_LIMIT = 200
+PORT_HTTP_MAX_CONNECTIONS_LIMIT = 100
```

Comment:
Wonder if it's worth having this configurable by env var for quick patches/tests later on